### PR TITLE
tikzit: Add version 2.1.6

### PR DESCRIPTION
**Pull request by adrianocorrea**

### CODE REVIEW

File: bucket/tikzit.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "2.1.6",
+    "description": "Graphical tool for rapidly creating graphs and string diagrams using PGF/TikZ",
+    "homepage": "https://tikzit.github.io/",
+    "license": "AGPL-3.0-or-later",
+    "architecture": {
```
Comment (Ash258) [inline bucket/tikzit.json:6]: Add note about reg files 
image

File: bucket/tikzit.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "64bit": {
```
Comment (Ash258) [inline bucket/tikzit.json:7]: Where is stated it is 64 bit??

image

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by adrianocorrea — COMMENTED** [review]
Added note about reg files and fixed architecture

File: bucket/tikzit.json (hunk 1 above)
Comment (adrianocorrea) [reply bucket/tikzit.json]: Done.

File: bucket/tikzit.json (hunk 2 above)
Comment (adrianocorrea) [reply bucket/tikzit.json]: Fixed.

**Review by Ash258 — APPROVED** [review]

**Pull request merged**